<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileControlWidget</class>
 <widget class="QWidget" name="FileControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>128</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileControlWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_Open">
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_Save_As">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>另存为...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_Save">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_Close">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
